doc_admin/AdminManual.adoc

```diff
@@ -650,55 +659,75 @@ Default configuration is available in the _settings.properties.dist_ file:
 
 [source,properties]
 ----
+###########################################
 # This file contains Open Hospital settings
+###########################################
+# external settings
+MODE=OH_MODE
+###########################################
```

Comment:
To reflect new one in gui https://github.com/informatici/openhospital-gui/pull/1682/files#diff-1a8920a6e31a618e733a43e60b8e3da5c79b300b1756afaeab07ee953abc0ec5

Reply:
Fixed.